var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -317,6 +324,11 @@ class Trilinos(CMakePackage):
     conflicts('+adios2', when='@:12.14.1')
     conflicts('+adios2', when='@xsdk-0.2.0')
     conflicts('+pnetcdf', when='~netcdf')
+    conflicts("+wrapper", when="~cuda")
+    conflicts("+wrapper", when="%clang")
```

Comment:
Shouldn't there be a conflict if the wrapper is not being used, but cuda is enabled and the compiler is not clang?

Reply:
what's the correct spack syntax for `when='~%clang'`?

Reply:
There isn't one. If `kokkos-nvcc-wrapper` is incompatible with clang then the restriction should be moved down into that.

Reply:
OK, I guess there is sensible way to implement @amklinv-nnl 's suggestion then